src/network/netdev/xfrm.c

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <net/if.h>
+
+#include "netdev/xfrm.h"
+
+static int xfrm_config_verify(
+    NetDev *netdev,
+    const char *filename
+) {
+    Xfrm *x;
+
+    assert(netdev);
+    assert(filename);
+
+    x = XFRM(netdev);
+
+    if (!x->dev) {
+        log_warning("Xfrm NetDev without dev set in %s. Defaulting to 'lo'", filename);
+        if(!((x->dev = strdup("lo")))) {
+            log_error("Could not allocate memory");
+            return -ENOMEM;
+        }
+    }
+
+    if (!x->if_id) {
+        log_warning("Xfrm NetDev without if_id set in %s. Defaulting to '0'", filename);
```

Comment:
please be less cryptic. not everyone has a degree in advanced networkology. For example I don't, and I really don't know what "if_id" is even supposed to mean.